src/ElectricPokemon.java

```diff
@@ -0,0 +1,89 @@
+public class ElectricPokemon extends Pokemons {
+    //variabelen
+    boolean solar;
+    boolean gas;
+    boolean wind;
+
+    //constructs
+    public ElectricPokemon() {
+    }
```

Comment:
Je moet je afvragen of je in dit geval wil dat je een leeg electic object kan aanmaken.